src/mobile/components/layout/templates/main.jade

```diff
@@ -3,13 +3,15 @@ include ./login-signup
 
 block body
   #content
-    block banner
-      mixin login-signup("Header")
-
-    header
-      != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, notificationCount: sd.NOTIFICATION_COUNT })
+    #header-content
+      block banner
+        mixin login-signup("Header")
+      header
+        != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, notificationCount: sd.NOTIFICATION_COUNT })
 
     main
+      mixin login-signup("Header")
```

Comment:
Duplicated here to push content down the correct amount w/o needing javascript (is hidden via CSS above).